tests/api/RestfulServiceTest.php

```diff
@@ -196,38 +196,11 @@ public function testHttpErrorWithoutCache() {
 	public function testHttpErrorWithCache() {
 		$subUrl = 'RestfulServiceTest_Controller/httpErrorWithCache';
 		$connection = new RestfulServiceTest_MockErrorService(Director::absoluteBaseURL(), 0);
-		$this->createFakeCachedResponse($connection, $subUrl);
 		$response = $connection->request($subUrl);
 		$this->assertEquals(400, $response->getStatusCode());
-		$this->assertEquals("Cache response body",$response->getCachedBody());
+		$this->assertEquals(false,$response->getCachedBody());
 		$this->assertContains("<error>HTTP Error</error>", $response->getBody());
-
-	}
-
-	/**
-	 * Simulate cached response file for testing error requests that are supposed to have cache files
-	 *
-	 * @todo Generate the cachepath without hardcoding the cache data
-	 */
-	private function createFakeCachedResponse($connection, $subUrl) {
```

Comment:
This shouldn't be removed, it should simply be creating the cache in a different way.